java-frontend/src/main/java/org/sonar/java/se/ExplodedGraph.java

```diff
@@ -113,7 +117,7 @@ public Tree syntaxTree() {
     public final ProgramPoint programPoint;
     @Nullable
     public final ProgramState programState;
-    private final List<Node> parents;
+    private final Map<Node, MethodYield> parents;
```

Comment:
Ok, after further investigation, there is still some cases where multiple yields can conduct to the same node, when considering variadic methods.